feature-libs/asm/occ/adapters/occ-asm.adapter.ts

```diff
@@ -65,4 +68,27 @@ export class OccAsmAdapter implements AsmAdapter {
       .get<CustomerSearchPage>(url, { headers, params })
       .pipe(this.converterService.pipeable(CUSTOMER_SEARCH_PAGE_NORMALIZER));
   }
+
+  bindCart({ cartId, customerId }: BindCartParams): Observable<unknown> {
+    const headers = InterceptorUtil.createHeader(
+      USE_CUSTOMER_SUPPORT_AGENT_TOKEN,
+      true,
+      new HttpHeaders()
+    );
+    let params: HttpParams = new HttpParams()
+      .set('baseSite', this.activeBaseSite)
+      .set('cartId', cartId)
+      .set('customerId', customerId);
+
+    const url = this.occEndpointsService.buildUrl(
+      'asmBindCart',
+      {},
+      {
+        baseSite: false,
+        prefix: false,
+      }
+    );
+
+    return this.http.post<void>(url, {}, { headers, params });
```

Comment:
please consider to have this `catchError((error) => throwError(normalizeHttpError(error)))`, so the caller can the error. "Cart failed to be assigned" is generic.